### ISSUE TITLE
Fix patient app colors for 508 compliance

### ISSUE THREAD

**Issue by drdavec** [chronic-care/mcc-project#310]
We need to prioritize this task, which appears to be a relatively easy change, e.g. in the CSS to modify either background color or link color to increase contrast.  This is needed to provide a report to VA for app registration to show due diligence for 508 compliance. 

This message from Matt at OHSU:  I just ran the Axe chrome extension tool against the MyCarePlanner app to identify Section 508 compliance.  It appears that there is really only one issue that comes up, flagged “serious,” that I think would be a very easy thing to fix and might get us over this hurdle.  Granted, 508 compliance isn’t strictly required, only strongly recommended, so we don’t *need* to do it, but I think we’ll be in a better place all around if we can check that box.
 
See https://www.deque.com/section-508/ for details on Section 508 and WCAG by the makers of the Axe tool.
 
Here’s a screenshot of the issue as documented by Axe:
 
(https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBc0srIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--55fc579df0bfd1910b2e8e05fddc966f9c44036e/unnamed.png)
 
Basically, there is a particular CSS style that’s being used in the app that employs the color blue (#007bff), which doesn’t provide sufficient contrast against a pure white background (#ffffff).  It’s almost there, providing a contrast of 3.97:1, when it needs to be 4.5:1.
 
Can this color blue be made a bit darker?  I think that might be all that’s needed.[Private Zenhub Image]

**Comment by savanahmueller** [comment]
Current blue: #007BFF (result 3.97:1)
New blue: #0071EB (result 4.6:1)
